src/ensembl/src/global/globalConfig.ts

```diff
@@ -9,6 +9,11 @@ export enum BreakpointWidth {
   BIG_DESKTOP = 1800
 }
 
+export enum DisplayType {
+  FULL = 'full',
+  COMPACT = 'compact'
+}
```

Comment:
That doesn't belong to the global config; it's a very local setting of `BrowserNavBarMain`

Reply:
It is for now, but this looks like an `enum` that we can easily reuse elsewhere. Hence, why I put it in `globalConfig`.

Reply:
Oh-uh.
Is it the part of the [Status type](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/shared/types/status.ts) then?

_(Although I still think they belong in `BrowserNavBarMain` component; they don't even leak out of it)_

Reply:
[`TrackPanel`](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/content/app/browser/track-panel/TrackPanel.tsx#L45) is another example of this. And then there is the [`BrowserBar`](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/content/app/browser/browser-bar/BrowserBar.tsx#L108) too.

Reply:
@msmanoj do you have any opinion about this type?

Reply:
As discussed, we can revert this to use the `breakpointwidth` within the child component to show/hide elements until we have a clear idea of how `DisplayType` needs to work.
Also, we agreed to change the prop name `breakpointwidth` to `viewportwidth` instead.

Reply:
:white_check_mark: